front/app/containers/Admin/messaging/texting/ViewCreatedMessage/index.tsx

```diff
@@ -0,0 +1,217 @@
+import React, { useState, useEffect } from 'react';
+
+// components
+import TextArea from 'components/UI/TextArea';
+import Button from 'components/UI/Button';
+import { Label, Box, IconTooltip } from '@citizenlab/cl2-component-library';
+import { Section, SectionField } from 'components/admin/Section';
+import HelmetIntl from 'components/HelmetIntl';
+import TextingHeader from '../components/TextingHeader';
+import FormattedStatusLabel from '../components/FormattedStatusLabel';
+
+// i18n
+import { FormattedDate } from 'react-intl';
+
+// utils
+import clHistory from 'utils/cl-router/history';
+import { withRouter, WithRouterProps } from 'react-router';
+
+// hooks
+import useTextingCampaign from 'hooks/useTextingCampaign';
+
+// services
+import {
+  updateTextingCampaign,
+  ITextingCampaignData,
+  ITextingCampaignStatuses,
+} from 'services/textingCampaigns';
+
+// styling
+import styled from 'styled-components';
+import { isNilOrError } from 'utils/helperUtils';
+
+const StyledForm = styled.form`
+  width: 500px;
+`;
+
+// enough to fit 3 messages, actual functionality TBD in subsequent ticket
+const MAX_CHAR_COUNT = 480;
+
+const getTitleMessage = (campaignStatus: ITextingCampaignStatuses) => {
+  switch (campaignStatus) {
+    case 'draft':
+      return 'Draft SMS Campaign';
+    case 'sending':
+      return 'Sending SMS Campaign';
+    case 'sent':
+      return 'Sent SMS Campaign';
+    case 'failed':
+      return 'Failed SMS Campaign';
+    default:
+      return 'Created SMS Campaign';
+  }
+};
+
+const getAdditionalInfoByStatus = (campaign: ITextingCampaignData) => {
+  switch (campaign.attributes.status) {
+    case 'draft':
+      return (
+        <>
+          Created at: <FormattedDate value={campaign.attributes.created_at} />
+        </>
+      );
+    case 'sending':
+      return (
+        <>
+          Sent at: <FormattedDate value={campaign.attributes.sent_at} />
+        </>
+      );
+    case 'sent':
+      return (
+        <>
+          Sent at: <FormattedDate value={campaign.attributes.sent_at} />
+        </>
+      );
+    case 'failed':
+      return (
+        <>
+          Created at: <FormattedDate value={campaign.attributes.created_at} />
+        </>
+      );
+    default:
+      return (
+        <>
+          Created at: <FormattedDate value={campaign.attributes.created_at} />
+        </>
+      );
+  }
+};
+
+const ViewCreatedMessage = (props: WithRouterProps) => {
+  const [inputPhoneNumbers, setInputPhoneNumbers] = useState('');
+  const [inputMessage, setInputMessage] = useState('');
+  const [remainingChars, setRemainingChars] = useState(MAX_CHAR_COUNT);
+
+  const { campaignId } = props.params;
```

Comment:
I think it's clearer to have the prop overview if we extract at line 90. Code style preference, so feel free to ignore.